scripts/src/main/resources/scripts/command/ide

```diff
@@ -63,6 +63,10 @@ function doUpdateScripts() {
       exit 0
     fi
   fi
+  local current_version
+  current_version="$("${DEVON_IDE_HOME}/scripts/devon" -v)"
+  doEcho "\nRun migration scripts to version ${target_version} ..."
+  doDevonCommand migration "${current_version}"
```

Comment:
One more thing. In the refinement meeting when introducing the issue story, I showed that on windows due to file locking we have to do the update asynchronous.
So this will most probably not work on windows. See here:
https://github.com/devonfw/ide/blob/c53d140ccc881b869a9e082a599f25a9379e7320/scripts/src/main/resources/scripts/functions#L779

Reply:
Could we also run the the migrate script for windows like this? 
 `(sleep 10;doBackup "${target}" "${3}";doRunCommand "mv '${1}' '${target}'";doRunCommand "${DEVON_IDE_HOME}/scripts/migration" "${current_version}") & `
 
 I don't see another option as we have to run the migration scripts after the `backup` function.

Reply:
Yes, you found the perfect solution and implemented it already. Thanks. So I am resolving this.